### ISSUE TITLE
[Feature request]: Subtitles

### ISSUE THREAD

**Issue by kitunas** [vzhd1701/gridplayer#194]
### Suggestion

Can you add option to turn on Subtitles? I have video files with subtitles available automatically when I play it on VLC but they do not show up when played in Gridplayer

**Comment by joshinils** [comment]
As an alternative, you can always use FFmpeg and burn in the subtitles, then play the file(s) with the burned in subtitles.

**Comment by Master-Of-Shadows** [comment]
I want to join this request. Video files already have subtitle streams so burning them in is nonsense.

**Comment by frasu** [comment]
> as an alternative, you can always use ffmpeg and burn iun the subtitles, then play the file(s) with the burned in subtitles.

Yes, but this is my case. I want to open the same file 4 times and 4 times with different subtitles to check it simultaneously. Then decide what is the best option, correct it and finally burn it.

**Comment by m4heshd** [comment]
Dang, this has been open since 2023? Guess it's not happening. This feature would've been so useful to me right now.
